var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -105,6 +105,8 @@ class Paraview(CMakePackage, CudaPackage):
     depends_on('double-conversion')
     depends_on('expat')
     depends_on('eigen@3:')
+    conflicts('target=ppc64le', when='@3.3.0:3.3.4')
```

Comment:
I think you mean:
```suggestion
    conflicts('target=ppc64le:', when='@3.3.0:3.3.4')
```
Otherwise what is written here will rule out the generic architecture but allow more specific ones e.g. `power9le`.

Reply:
Thanks for catching this, I updated the PR. Can you explain this "`:`" mechanism a little? I read the docs on `conflicts` but didn't see explicit reference to the `:`.

Reply:
@qwofford Sure. There are two parts to it:

1. Spack can distinguish microarchitectures at a finer level than just the base ISA (like `ppc64le`, or `x86_64` etc.). We rely on an external Python package that originated from Spack (https://github.com/archspec/archspec) for that part. Each architecture family defines a graph.
2. In directives you can use the same range semantic as with versions to specify an open or closed range of targets. For instance `ppc64le:` means anything from the base architecture up, so it covers `ppc64le`, `power8le` and `power9le`. You can also do more exotic things like `haswell:skylake`, which would cover `haswell`, `broadwell` and `skylake`...  but I never saw anything like this used in real packages.

Let me know if I was clear enough in the explanation :slightly_smiling_face:

Reply:
Yes! This is clear. In fact, I think I saw this presented at CANOPIE and I'm just now connecting the dots. Thanks!